README_PROCESS.md

```diff
@@ -0,0 +1,221 @@
+## Repository branch structure
+
+The SYCL specification intends to have periodic major versions, where each is
+named according to the year in which it is approved by the working group.
+Although there were previous major versions, this repository starts with the
+SYCL 2020 specification.
+From this point forward, the SYCL specification is "unified", meaning that a
+single document serves as the official specification for all major versions
+from SYCL 2020 onwards.
+
+There are often two active branches in the repository: a branch named "main" and
+a branch whose name has the form "sycl-xxxx" (e.g. "sycl-2020").
+The "main" branch always exists, and it represents the most current version of
+the specification.
+Depending on the release cycle, "main" may represent the major version that is
+currently published, or it may represent a future major version that is not yet
+published.
```

Comment:
I'm a little confused by this.

It sounds like you're saying that "main" might contain:
- A specification that has been ratified and published, and nobody's made any changes to it yet; or
- A specification that hasn't been ratified or published.

...but I'm not sure I understand why this is relevant to GitHub users, so I think I might have misunderstood something. It seems like we wouldn't be in the first situation for very long.

Reply:
Based on your comments, I think I did not explain the branch structure well.  I tried rewriting the first section in db4aeec94efa742077b2a2c0e0ea3fe787d8473c.  Does the new introduction make it clearer?